### PR TITLE
Improve devise views

### PR DESCRIPTION
**Pull request by double-beep**
- Add the option to edit account settings in the dropdown
- Allow users to deauthorize from Sentinel. Admins *shouldn't* be the only ones to do this.

### CODE REVIEW

File: app/controllers/se_auth_controller.rb [hunk 1]
```diff
@@ -46,12 +46,14 @@ def already_done
 
   def deauth
```
Comment (ArtOfCode-) [inline app/controllers/se_auth_controller.rb:47]: You've forgotten to remove the `before_action :verify_admin`, so this won't work. In any case, users shouldn't be able to delete the whole `stack_user` record; allow admins to do that, but a user-triggered deauth should just remove the SE API token.

Comment (double-beep) [reply app/controllers/se_auth_controller.rb:47]: If the access token is removed, how are users going to regenerate a new one?

Comment (ArtOfCode-) [reply app/controllers/se_auth_controller.rb:46]: If they're de-authorizing the app, why would they _want_ to generate a new one?

Comment (double-beep) [reply app/controllers/se_auth_controller.rb:46]: Because they wanted to or accidentally invalidated one?

Comment (ArtOfCode-) [reply app/controllers/se_auth_controller.rb:46]: I don't see where you're going. The deauth button is there to remove details from the app if someone doesn't want to use it any more. If that's the case, there's no need to be able to regenerate a token.

Comment (double-beep) [reply app/controllers/se_auth_controller.rb:46]: Well, in most of the apps/userscripts I use, users always have the option to deauthorize and then authorize again. OTOH, you may be right - should I revert the commit?

Comment (ArtOfCode-) [reply app/controllers/se_auth_controller.rb:46]: Oh, I see what you're aiming at. No, this is good as-is in that case.

**Pull request merged**